templates/flowimages/flowimage_filter.haml

```diff
@@ -1,3 +0,0 @@
--extends "flowimages/flowimage_list.html"
```

Comment:
Don't we need this file anymore?

Reply:
We don't need this file because we use `flowimage_list.html` instead of it.